### PR TITLE
[FEATURE] Add environment tests

### PR DESCRIPTION
**Pull request by serhiislobodian**

### CODE REVIEW

**Comment by lverba** [comment]
ok to test

**Comment by lverba** [comment]
retest this please

File: preload.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ const settings = electron.remote.require("./server/settings");
 const ipcTunnelClient = new IpcClient();
 
 // Constants
+window.env = process.env;
```
Comment (boodyvo) [inline preload.js]: Do not push all env to front, push only necessary

File: test/setup.js [hunk 2]
```diff
@@ -16,6 +16,7 @@ const ipcClient = sinon.stub();
 window.ipcClient = ipcClient;
 
 // Constants
+window.env = process.env;
```
Comment (boodyvo) [inline test/setup.js]: Do not push all env to front, push only necessary

**Pull request merged**